pipelines/convert_inferencedata_to_parquet.R

```diff
@@ -28,10 +28,10 @@ tidy_and_save_mcmc <- function(
   good_chain_tol
 ) {
   model_dir <- path(model_run_dir, model_name)
-  inference_data_path <- path(model_dir, "inference_data", ext = "csv")
+  inference_data_path <- path(model_dir, "inference_data", ext = "parquet")
 
   tidy_inference_data <- inference_data_path |>
-    read_csv(show_col_types = FALSE) |>
+    read_parquet(show_col_types = FALSE) |>
```

Comment:
Is the `show_col_types` arg still required?
```suggestion
    read_parquet() |>
```

Reply:
Good catch!